app/src/main/res/values/strings.xml

```diff
@@ -10,12 +10,25 @@
     <string name="serial_number">Serial Number</string>
     <string name="ip_address">IP Address</string>
     <string name="start_payment">Start Payment</string>
+    <string name="start_refund">Refund</string>
+    <string name="change_payment_type">Type</string>
 
     <string name="amount_hint">10.99</string>
     <string name="currency_hint">EUR</string>
 
     <string name="state_idle_status">Idle</string>
     <string name="state_in_progress_status">In Progress</string>
-    <string name="state_complete_status">Payment completed</string>
+    <string name="state_complete_status">Transaction completed</string>
     <string name="state_error_status_format">An error has occurred.\n%s</string>
+    <string name="title_activity_transaction">TransactionActivity</string>
+    <!--
+    This string is used for square devices and overridden by hello_world in
+    values-round/strings.xml for round devices.
+    -->
+    <string name="hello_world">Hello Square World!</string>
+
+    <string-array name="transaction_types">
+        <item>Payment</item>
+        <item>RefundToACard</item>
```

Comment:
Using resources for Fragment names is not a good idea, see explanation in Fragment.